spec/jobs/import_url_job_spec.rb

```diff
@@ -26,6 +31,10 @@
     stub_request(:get, "http://example.org#{file_hash}").to_return(
       body: File.open(File.expand_path(file_path, __FILE__)).read, status: 200, headers: response_headers
     )
+
+    allow(BrowseEverything::Retriever).to receive(:new).and_return(mock_retriever)
+    allow(mock_retriever).to receive(:retrieve)
+    allow(HTTParty).to receive(:head).and_return(mock_http_result)
```

Comment:
Should we use WebMock instead of mocking `HTTParty` directly? We are [requiring it in the test suite](https://github.com/samvera/hyrax/blob/51b45cc2dc35eb8101f3eec5bde28922b63a0872/spec/spec_helper.rb#L60) (but maybe not actually using it), and it seems like it will be more robust than coupling the test to the specific client software.

Reply:
@no-reply - The test has been updated to use WebMock instead of mocking HTTParty.  Could you review again.  Thanks